manifests/sysconfig_init.pp

```diff
@@ -45,6 +45,17 @@
   Boolean              $autoswap          = false,
 ) {
 
+  if 'systemd' in $facts['init_systems'] {
+    $_unit_file_content = @("END")
+      [Service]
+      ExecStart=-/bin/sh -c "${single_user_login}; /usr/bin/systemctl --fail --no-block default"
+      | END
+
+    systemd::unit_file { ['emergency.service', 'rescue.service']:
```

Comment:
I'm not sure what this does.  Is this to permit the new user to log in during single user mode?

Reply:
@op-ct It's the single user login mechanism. The old method of using `SINGLE` in the `sysconfig` file probably doesn't do what you want (but is still documented so I left it).

Reply:
Good to know; thanks!